src/index.html

```diff
@@ -11,6 +11,29 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form class = "search_form" data-qa="big">
+      <img
+        src = "./images/Search.svg"
+        class = "search_ikon"
+        alt="Search"
```

Comment:
```suggestion
        src="./images/Search.svg"
        class="search_ikon"
        alt="Search"
```
почему одни атрибуты с пробелами, а другие без?